### PR TITLE
Feat/rl 6170243374/va ventures page

### PR DESCRIPTION
**Pull request by RyanCrux**
## What type of PR is this? (check all applicable)

- [X] Feature
- [ ] Bug Fix
- [ ] Hot Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
code VA Ventures - Landing Page

## Related Tickets & Documents
- Monday Task: 
https://blackberg-group.monday.com/boards/1931719392/pulses/6170243374

Design Ticket: https://blackberg-group.monday.com/boards/1931719392/pulses/6170242636

Figma: https://www.figma.com/design/gCLe2fX5MaECTIhWx4g8EC/VA-Ventures?node-id=2006-1415&t=5iSb3PPn43G5XneM-0

Dev Sandbox: https://www.dev-innovation.bgsandbox.com/ecosystem/views/national-centers/va-ventures/

## QA Instructions, Screenshots, Recordings

Inspect new page. compare against figma. Check links

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Checked with [Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/ventures/1_masthead.html
 - _includes/ventures/2_mission.html
 - _includes/ventures/3_news.html
 - _includes/ventures/4_director.html
 - _includes/ventures/5_connect.html
 - _site/assets/img/ventures/Picture2
 - 1.png
 - _site/assets/img/ventures/bubble-lines-sm.svg
 - _site/assets/img/ventures/bubble-lines.svg
 - _site/assets/img/ventures/iaquinto.png
 - _site/assets/img/ventures/levin.png
 - _site/assets/img/ventures/masthead-img.png
 - _site/assets/img/ventures/ramirez.png
 - _site/assets/img/ventures/richburg.png
 - _site/assets/img/ventures/ripley.png
 - _site/assets/img/ventures/test.png
 - _site/assets/img/ventures/vezeau.png
 - _site/assets/img/ventures/vha-ventures-logo.svg
 - _site/assets/img/ventures/willis.png
 - _site/assets/js/ideas-home-news
 - copy.js
 - _site/assets/js/ventures-home-news.js
 - assets/img/ventures/bubble-lines-sm.svg
 - assets/img/ventures/bubble-lines.svg
 - assets/img/ventures/iaquinto.png
 - assets/img/ventures/levin.png
 - assets/img/ventures/masthead-img.png
 - assets/img/ventures/ramirez.png
 - assets/img/ventures/richburg.png
 - assets/img/ventures/ripley.png
 - assets/img/ventures/vezeau.png
 - assets/img/ventures/vha-ventures-logo.svg
 - assets/img/ventures/willis.png
 - assets/js/ventures-home-news.js
 - _site/va-ventures/index.html
 - assets/js/ideas-home-news.js
 - va-ventures/index.html

### CODE REVIEW

File: _site/assets/img/ventures/Picture2 1.png
Comment (ecstech-admin) [inline _site/assets/img/ventures/Picture2 1.png:1]: Remove old/test files from repo

File: _site/assets/img/ventures/test.png
Comment (ecstech-admin) [inline _site/assets/img/ventures/test.png:1]: Remove old/test files from repo

File: _site/assets/js/ideas-home-news copy.js
Comment (ecstech-admin) [inline _site/assets/js/ideas-home-news copy.js:1]: Remove old/test files from repo

**Review by ecstech-admin — COMMENTED** [review]
Looks great! Nothing too major

Should probably add meta description
rm extra files from repo
"Blue Team" title should be #E42026
Text for "Leadership Team" section should be aligned left on mobile
image

**Review by will-bbg — APPROVED** [review]
🚀

**Pull request merged**